Cargo.toml

```diff
@@ -44,10 +55,6 @@ gfx_window_sdl = { path = "src/window/sdl", version = "0.4" }
 gfx_device_dx11 = { path = "src/backend/dx11", version = "0.4" }
```

Comment:
Do we have an issue open on cargo to allow for default features depending on target? Useful for users